src/main/java/de/zalando/aruha/nakadi/domain/EventType.java

```diff
@@ -2,30 +2,60 @@
 
 import java.util.List;
 
+import com.fasterxml.jackson.annotation.JsonGetter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonSetter;
 import com.google.common.collect.Lists;
 
 public class EventType {
```

Comment:
I wonder what was the problem here, as for me all the fields except 'eventTypeSchema' should be handled well by default jackson deserializer. Also, you can put annotation directly on fields.